### PR TITLE
Removed copy from project duplication

### PR DESCRIPTION
**Pull request by mebverse**

### CODE REVIEW

File: how-to/duplicate-project.rst [hunk 1]
```diff
@@ -32,37 +32,30 @@ or in the project view:
...
 
-A fork is similar to a copy, with the distinction that the new project retains the original's Git history in its
-codebase.
+A fork is a form of duplication in which the new project will be an exact but independent copy of the original. A fork
+will include the original’s code, database content, media, environment variables and all branches,  and retains the Git
+history in its codebase. 
 
-A fork is therefore useful when you want a duplicate in order to undertake substantial new development, as it allows you
-to merge back changes from the duplicate into the original using Git.
+A fork is therefore a useful tool when you want to duplicate in order to undertake substantial new development, as it
```
Comment (evildmp) [inline how-to/duplicate-project.rst]: _A fork is therefore a useful tool_ - it's not really a tool. 

Maybe just:

```
A fork is therefore useful when you want to undertake substantial new development...
```

**Review by evildmp — CHANGES_REQUESTED** [review]

File: how-to/duplicate-project.rst (hunk 1 above)
Comment (mebverse) [reply how-to/duplicate-project.rst]: Done

**Review by evildmp — APPROVED** [review]
Ready to be merged as soon as the change has been deployed on the live Control Panel.

File: how-to/duplicate-project.rst [hunk 2]
```diff
@@ -32,37 +32,30 @@ or in the project view:
...
-
 
 Fork
 ^^^^
 
-A fork is similar to a copy, with the distinction that the new project retains the original's Git history in its
-codebase.
+A fork is a form of duplication in which the new project will be an exact but independent copy of the original. A fork
+will include the original’s code, database content, media, environment variables and all branches,  and retains the Git
+history in its codebase.
```
Comment (evildmp) [inline how-to/duplicate-project.rst:49]: Are you certain that a fork contains the environment variables of the orginal?

Comment (mebverse) [reply how-to/duplicate-project.rst:49]: I checked it, yes it contains

**Pull request merged**